prototypes/python/otel.py

```diff
@@ -0,0 +1,127 @@
+
+import logging
+from typing import List, Sequence, Tuple
+from pkg_resources import iter_entry_points
```

Comment:
`pkg_resources` has been deprecated (see [this](https://github.com/open-telemetry/opentelemetry-python/pull/3047), and also [this](https://github.com/open-telemetry/opentelemetry-python/pull/3168)).

Reply:
Ah good to know. It's probably ok for the prototype but wouldn't want to ship this code :D